server/game/core/card/baseClasses/PlayableOrDeployableCard.ts

```diff
@@ -41,7 +40,9 @@ export class PlayableOrDeployableCard extends Card {
     }
 
     public override getActions(): PlayerOrCardAbility[] {
-        return this.defaultActions.concat(super.getActions());
+        const actions = this.defaultActions.concat(super.getActions());
+
+        return actions;
```

Comment:
Why split into two lines?

Reply:
There was something in the middle originally. I've made it more concise